src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1366,7 +1366,7 @@ private void visitFunctionParameters(FunctionType ftype, boolean emitTemplatized
         Iterable<Node> parameterNodes = functionSource.getFirstChild().getNext().children();
         names = Iterables.transform(parameterNodes, NODE_GET_STRING).iterator();
       }
-      char pName = 'a'; // let's hope for no more than 26 parameters...
+      char pName = 'a'; // let's hope for no more than 52 parameters...
```

Comment:
I thought this is pretty clever extension, without doing the full work :) But sure I will do the little interview problem to implement the proper name.next() :)